src/styles/index.scss

```
@@ -1,3 +1,46 @@
-body {
+@import 'header';
+@import 'card';
+@import 'stars';
+@import 'variables';
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.catalog {
+  padding: 50px 40px;
+  max-width: 944px;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (max-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
```

It's scss, you can make it like that
```suggestion
  gap: 46px 48px;
  justify-content: center;
}

@media (max-width: 488px) {
  .catalog {
    grid-template-columns: repeat(1, 200px);

       @media (min-width: 488px) {
           grid-template-columns: repeat(1, 200px);
       }
```